src/components/elements/Inputs/Checkbox/Checkbox.tsx

```diff
@@ -16,68 +17,68 @@ export interface CheckboxProps extends Omit<React.InputHTMLAttributes<HTMLInputE
   inputClassName?: string;
 }
 
-const Checkbox = ({
-  label,
-  required,
-  error,
-  className,
-  disabled,
-  defaultChecked,
-  textClassName,
-  form,
-  inputClassName,
-  ...props
-}: CheckboxProps) => {
-  const id = useId();
+const Checkbox = forwardRef<HTMLInputElement, CheckboxProps>(
+  (
+    { label, required, error, className, disabled, defaultChecked, textClassName, form, inputClassName, ...props },
+    ref
+  ) => {
+    const id = useId();
 
-  const labelIsComponent = label && typeof label !== "string";
+    const labelIsComponent = label && typeof label !== "string";
 
-  return (
-    <div
-      className={classNames(className, `relative flex items-start gap-4`, {
-        "justify-between": !className?.includes("justify")
-      })}
-    >
-      {label && (
-        <label htmlFor={id} className="w-full">
-          <If condition={labelIsComponent}>
-            <Then>{label}</Then>
-            <Else>
-              <Text
-                as="label"
-                htmlFor={id}
-                variant="text-body-600"
-                className={`max-w-[365px] ${textClassName}`}
-                containHtml
-              >{`${label} ${required ? "*" : ""}`}</Text>
-            </Else>
-          </If>
-        </label>
-      )}
-
-      <input
-        {...props}
-        {...form?.register(props.name)}
-        id={id}
-        defaultChecked={defaultChecked}
-        required={required}
-        disabled={disabled}
-        type="checkbox"
-        className={twMerge(
-          classNames(
-            "h-5 w-5 cursor-pointer rounded-sm border-neutral-400 transition-all duration-300 checked:text-primary-500 focus:ring-transparent disabled:bg-neutral-400",
-            { "absolute opacity-0": labelIsComponent }
+    return (
+      <div
+        className={
+          (tw(
+            classNames(`relative flex items-center gap-4`, {
+              "justify-between": !className?.includes("justify")
+            })
           ),
-          inputClassName
+          className)
+        }
+      >
+        {label && (
+          <label htmlFor={id} className="w-full">
+            <If condition={labelIsComponent}>
+              <Then>{label}</Then>
+              <Else>
+                <Text
+                  as="label"
+                  htmlFor={id}
+                  variant="text-body-600"
+                  className={`max-w-[365px] ${textClassName}`}
+                  containHtml
+                >{`${label} ${required ? "*" : ""}`}</Text>
```

Comment:
This text should use translation. I think just wrapping `label` in `t()` from `useT` should be OK here. Sometimes you'd want to include the decoration (`*`) so the translators can make sure it's in the right place in other languages, but I think just leaving it on the end is OK in this case.

Reply:
done